metrics-core/src/main/java/io/dropwizard/metrics/CsvReporter.java

```diff
@@ -126,16 +134,18 @@ public CsvReporter build(File directory) {
                                    rateUnit,
                                    durationUnit,
                                    clock,
-                                   filter);
+                                   filter,
+                                   csvFileProvider);
         }
     }
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CsvReporter.class);
-    private static final Charset UTF_8 = Charset.forName("UTF-8");
+    private static final Charset UTF_8 = Charset.forName( "UTF-8" );
```

Comment:
There are a number of lines like this which haven't changed except for the insertion of some spaces. Could you fix these and make sure that all other method invocations don't contain spaces except after commas?